test/mapKeys.js

```diff
@@ -0,0 +1,9 @@
+var assert = require('assert');
+
+var R = require('..');
+
+describe('mapKeys', function() {
+  it('returns an object with each of the keys passed set to value returned for it by the given function', function() {
+    assert.deepEqual(R.mapKeys(R.toUpper, ['a', 'b', 'c']), {a: 'A', b: 'B', c: 'C'});
+  });
+});
```

Comment:
Honestly, I just modeled this after the test after the test for mapObj. I'll try to elaborate it a little bit.

By the way, thinking about the ongoing ES6-Map support I'm now also against the mapKeys naming, and would rather go with objFromKey proposed in the other bug (for similar reason, I think createMapEntry should be renamed).